jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```
@@ -142,4 +142,24 @@ DeleteComment=Are you sure that you want to delete comment: ''{0}'' ?
 DeleteCommentTitle=Delete comment confirmation 
 SortByDateDesc=Sort by date desc
 SortByDateAsc=Sort by date asc
+Bulk_Claim=Bulk Claim
+Claim_Confirmation=Are you sure that you want to claim the selected tasks?
+Claim_tasks=Claim selected tasks
+ClaimNotAllowedOn=Claim Task (id={0}, name=''{1}'') Not Allowed (The task is not Ready)
```

Just a suggestion to make the message less technical.
```suggestion
ClaimNotAllowedOn=Claim Task ''{1}'' (id={0}) not allowed, task is not in Ready state
```

---

updated